src/main/java/net/sacredlabyrinth/phaed/simpleclans/commands/data/Alliances.java

```diff
@@ -48,4 +48,17 @@ private void sendHeader() {
         chatBlock.setAlignment("l", "l");
         chatBlock.addRow("  " + headColor + lang("clan", sender), lang("allies", sender));
     }
+
+    protected void sendBlock() {
+        SettingsManager sm = plugin.getSettingsManager();
+        boolean more = chatBlock.sendBlock(sender, sm.getInt(PAGE_SIZE));
+
+        if (more) {
+            plugin.getStorageManager().addChatBlock(sender, chatBlock);
+            ChatBlock.sendBlank(sender);
+            ChatBlock.sendMessage(sender, sm.getColored(PAGE_HEADINGS_COLOR) + lang("view.next.page", sender,
+                    sm.getString(COMMANDS_MORE)));
+        }
+        ChatBlock.sendBlank(sender);
+    }
```

Comment:
You misunderstood the meaning of Push Down Method. 🍵 

> At first a certain method was meant to be universal for all classes but in reality is used in **only one subclass**. 
[Source](https://refactoring.guru/push-down-method)

As you see, `sendBlock` was used in 13 files, not in one subclass, right? 😅

Reply:
sendBlock() is not used in all the classes that are extending Sendable class such as ClanProfile and Lookup. These classes still have access to sendBlock() which is unnecessary hence I applied Push-Down Method. As the class is having access to more methods than is needed. 

Regarding the source you cited, I think they are trying to give an example and it is not a definition.

Reply:
sendBlock is used by 10 of the 12 subclasses. I would say there is reason for that method to exist there.
Also, your application of push-down created 10 blocks of duplicate code...

Either sendBlock() stays where it is or an interface can be created where the method is implemented.